sass/_aesop.scss

```diff
@@ -201,15 +220,26 @@
 	}
 }
 
-.aesop-article-chapter {
-    background-color: #333 !important;
+.aesop-article-chapter:after {
+    background-color: $color-dark-grey;
+    bottom: 0;
+    content: '';
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings